app/models/publication.rb

```diff
@@ -62,10 +62,14 @@ def self.publication_types
           where('published_on >= user_organization_memberships.started_on AND (published_on <= user_organization_memberships.ended_on OR user_organization_memberships.ended_on IS NULL)').
           distinct(:id) }
 
-  scope :subject_to_open_access_policy, -> { where('published_on >= ?', Publication::OPEN_ACCESS_POLICY_START) }
+  scope :subject_to_open_access_policy, -> { where("published_on >= ? AND publication_type ~* 'Journal Article'", Publication::OPEN_ACCESS_POLICY_START) }
 
```

Comment:
I _think_ you can use the new scope inside of this lambda too. For example, 
```ruby
-> { journal_article.where('published_on >= ?', Publication::OPEN_ACCESS_POLICY_START) }
```
... or did that not work?